native/packages/react-native-bpk-styles/src/shadows-test.js

```diff
@@ -0,0 +1,26 @@
+/*
+  * Backpack - Skyscanner's Design System
+  *
+  * Copyright 2017 Skyscanner Ltd
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *   http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+import shadows from './shadows';
+
+describe('shadows', () => {
+  it('should export the correct shadow styles', () => {
+    expect(shadows.base).toBeDefined();
+    expect(shadows.large).toBeDefined();
+  });
+});
```

Comment:
Could argue that this test is pointless - I was going to make it check the actual values, but then that's already covered by snapshot tests. I've left it in because it does no harm to have it, but I can remove it.

Reply:
Where is it covered by snapshot tests? You mean in the components that use those styles?